gap-analysis/index.md

```diff
@@ -455,9 +455,9 @@ Various proposals have been discussed for the syntax to be supported by the inte
 
 An advantage of this proposal is that it can be implemented using current technology without changes to other web standards, other than to specify that the attribute should be part of the accessibility tree built by browsers. While the functional syntax [‘name(arg1, ...)’]  represents a MathML-specific encoding, it is relatively easy to generate and to consume.
 
-The `intent` attribute is meant to be passed to AT. This allows AT to decide what is appropriate based on the needs/preferences of the reader. It also avoids the problem of having to know whether to embed in `aria-label` speech geared to someone who can't see the screen (and hence needs begin/end cues) and someone who can see the screen and needs audio reinforcement to help them decode what they see. It also leaves it to AT to add prosody cues and forcing a long 'a' sound, something that can't currently can't be done with `aria-label`.
+The `intent` attribute is meant to be passed to AT. This allows AT to decide what is appropriate, based on the needs/preferences of the reader. It avoids the problem of having to know whether to embed in `aria-label` speech geared to someone who can't see the screen (and hence needs begin/end cues) or geared to someone who can see the screen and needs audio reinforcement to help them decode what they see.
```

Comment:
Not that it really matters, but just so you know, the `bookmark` is the sync point and generates a callback to the caller. As used for speech, the callback initiates the highlighting [details encoded in the `mark` value] and hence happens *before* the text to speech, so it is probably technically clearer to put the bookmark on the same line before the text, although it does make it harder to see the text.

An `mi` with a single char is indeed the cue that MathPlayer uses to decide it should "spell" the contents. It might have some cleverness to do the same for roman numerals. Same for hex numbers in `mn` (non-digit/decimal/comma/space => speak each char).

*Currently* no AT that I know of looks for aria-label on math elements. But assuming we say they should and assuming we don't want to be in conflict with ARIA, when AT encounters `aria-label`, is should speak that as is and not look into the children. However, if we have a convention where '-' separates words, then on `intent`, AT would be able to see single chars and "spell" them.  I/we looked at hyphens, underlines and maybe one other connective earlier and found that none was safe to use across speech engines (or maybe it was AT -- they would get spoken in some circumstances). So something a spec for `intent` should probably say  is 'Hyphen is a word break and should be replaced by a space. After breaking, single letter characters should be spelled out to force them to spoken as they would by a person'. This does make me wonder, why are we using '-' and not a space? I'm sure there was a reason in the past, but I don't know if it is still valid.

Given the above, I'm not a fan of your rephrasing because the details of when AT might or might not spell something aren't the point, the point is that there is no option if `aria-label` is used; avoiding `aria-label` means AT has options which might include adding pauses, changing pitch (subscripts/superscripts), or rate. I hope I'm not misremembering, but I think as part of the ClearSpeak grant, we did an experiment with pitch changes and the kids didn't like them. None the less, other AT has used them -- I know Raman's seminal ASTER was a big fan of pitch changes.

Reply:
I made one more pass, phrasing it a little closer to the original, without running the risk of the previous vague broadness. In the end the reason *why* the prosody cues work with one attribute and not the other needs to be made clear, at least on a high level, or the reader is left wondering.

> This does make me wonder, why are we using '-' and not a space?

Because spaces look completely wrong in functional expressions. And from the very beginning we wanted to have the "names" we use in the values in a syntax where one could imagine them as either symbols or speech. Compare:

- `intent="binomial-coefficient($1, B-prime)"`
- `intent="binomial coefficient($1, B prime)"`

Reply:
Thanks for the reminder about why not use spaces.

Your change for this looks good. I'm ready to approve if you just clarify the "verbatim" sentence mentioned in the other comment.